packages/engine/lib/memory/Cargo.toml

```diff
@@ -16,14 +19,11 @@ shared_memory = { git = "https://github.com/hashdeps/shared_memory-rs" }
 thiserror = "1.0.31"
 tracing = "0.1.35"
 uuid = { version = "1.1.2", features = ["v4"] }
-# note: the `arrow-format` version must be kept in line with whatever `arrow2`
-# requires! Do not bump the version without checking this first.
-arrow-format = "0.6.0"
 num = "0.4.0"
 bytemuck = "1.10.0"
 
 [dev-dependencies]
-arrow2_convert = { git = "https://github.com/DataEngineeringLabs/arrow2-convert", features = [
+arrow2_convert = { git = "https://github.com/teymour-aldridge/arrow2-convert", branch = "update-arrow2", features = [
```

Comment:
When do we expect to use the upstream version for this?

Reply:
The linked PR is merged 🙂

Reply:
Can we also make sure we are sticking to a specific revision? Also, let's put the feature flags in the same line.

Reply:
Yes, let's wait for this!

Reply:
(It has been done)